crates/stackable-versioned-macros/src/lib.rs

```diff
@@ -223,7 +223,7 @@ pub fn versioned(attrs: TokenStream, input: TokenStream) -> TokenStream {
     // Module(ItemMod).
     let input = syn::parse_macro_input!(input as DeriveInput);
 
-    gen::expand(attrs, input)
+    codegen::expand(attrs, input)
```

Comment:
Now I have to make the same change in `protogen` :sweat_smile:

Reply:
I made the change, because in the future `gen` will be a reserved keyword and would require us to use `use crate::r#gen` for imports. Also, in doc comments `r#gen` didn't link to the item. I don't know if there is an upstream issue for that, instead I decided to rename the module to make that work again.